src/plugins/intel_npu/tests/functional/shared_tests_instances/execution_graph_tests/duplicate_inputs_outputs_names.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "execution_graph_tests/duplicate_inputs_outputs_names.hpp"
+
+#include "common/npu_test_env_cfg.hpp"
+#include "common/utils.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace ExecutionGraphTests;
+
+namespace {
+
+INSTANTIATE_TEST_SUITE_P(smoke_BehaviorTests,
```

Comment:
Although not under the `behavior/` directory, I've chosen the `smoke_BehaviorTests` in order to allow the test to fall under one of the validation jobs regexes.